VSRAD.Package/DebugVisualizer/Wavemap/WavemapView.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Windows.Media;
+
+namespace VSRAD.Package.DebugVisualizer.Wavemap
+{
+    public struct WaveInfo
+    {
+        public Brush BreakColor;
+        public uint BreakLine;
+        public int GroupIdx;
+        public int WaveIdx;
+    }
+
```

Comment:
It would be better to use `#pragma warning disable CA1815 ` here to turn off warning

Reply:
Thanks, resolved in b3bd8d0